live-examples/css-examples/basic-user-interface/outline-style.html

```diff
@@ -0,0 +1,44 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="outline-style">
+
+    <div class="example-choice">
+        <pre><code id="example_one" class="language-css">outline-style: none;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code id="example_two" class="language-css">outline-style: dotted;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_three" class="language-css">outline-style: solid;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_four" class="language-css">outline-style: groove;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_five" class="language-css">outline-style: inset;</code></pre>
```

Comment:
For both `inset` and `groove` it is very hard to see the styling as the contrast is very low. I would suggest adding `outline-color: #999;` to both this, and the `groove` examples.